lib/datadog/tracing/contrib/span_attribute_schema.rb

```diff
@@ -7,16 +7,18 @@ module Contrib
       module SpanAttributeSchema
         module_function
 
-        def fetch_service_name(env, default)
-          ENV.fetch(env) do
-            if Datadog.configuration.tracing.span_attribute_schema ==
-                Tracing::Configuration::Ext::SpanAttributeSchema::VERSION_ONE
-              Datadog.configuration.service
-            else
-              default
-            end
+        # rubocop:disable Style/OptionalArguments
+        def fetch_service_name(value = nil, default)
```

Comment:
Can you have a leading positional argument with a default value?

I'm confused on what `SpanAttributeSchema.fetch_service_name('my-value')` will do.

I guess Rubocop didn't like it either. Maybe it's a non-intuitive pattern.

Reply:
Great point 😄 

Done https://github.com/DataDog/dd-trace-rb/pull/2970/commits/c27577e592813e14faec446da2f391820d205c73